server/src/main/java/org/apache/iotdb/db/engine/StorageEngineV2.java

```diff
@@ -523,6 +528,34 @@ public void forceCloseAllProcessor() throws TsFileProcessorException {
     }
   }
 
+  public void closeStorageGroupProcessor(String storageGroupPath, boolean isSeq, boolean isSync) {
+    for (DataRegion dataRegion : dataRegionMap.values()) {
+      if (dataRegion.getLogicalStorageGroupName().equals(storageGroupPath)) {
+        for (TsFileProcessor tsFileProcessor : dataRegion.getWorkSequenceTsFileProcessors()) {
```

Comment:
The `unsequenceTsFileProcessors` is not considered here.

We need to select `getWorkSequenceTsFileProcessors()` or `getWorkUnSequenceTsFileProcessors()` according to `isSeq` here.

Reply:
fixed~